.github/workflows/test-lint-workflow.yml

```diff
@@ -23,6 +23,14 @@ jobs:
           WORKFLOW_LIST=$(ls .github/workflows | xargs -I {} echo -n ".github/workflows/{} ")
           echo "workflow-list=$WORKFLOW_LIST" >> $GITHUB_OUTPUT
 
+      - name: Intentional fail test for one line pipe block run
+        run: |
+          echo "This will fail"
+
+      - name: Success one line angle block indicator run
+        run: >
+          echo "This will not fail"
+
```

Comment:
@vgrassia @Eeebru  Is this correct here? Shouldn't it be that _any_ single line following the `run:` command fails the linter, requiring the two lines to be merged into a single line, starting with `run:`.

Example:
```yaml
- name: This is a test
  run: >
    echo "test"
```

Should be

```yaml
- name: This is a test
  run: echo "test"
```
However, this should be fine.

```yaml
- name: This is a test
  run: >
    echo "test"
    echo "another test"
```

Reply:
@mimartin12 Both this 

```
- name: This is a test
  run: >
    echo "test"
```
and this

```
- name: This is a test
  run: >
    echo "test"
    echo "another test"
```

Becomes a single line command separated by spaces unless it has more than 1 new line. https://yaml-multiline.info/

Reply:
@Eeebru  I understand that it works like that, however I am talking specifically about the styling we are attempting to enforce. In the past, we have enforced that all `run:` commands should be on a single line _IF_ it's just a single line command. 
Example:
We want to make sure that steps like this
```yaml
- name: This is a test
  run: >
    echo "test"
```
Are _instead_ this
```yaml
- name: This is a test
  run: echo "test"
```

With us enforcing that style, then both of these tests would fail.

```yaml

      - name: Intentional fail test for one line pipe block run
        run: |
          echo "This will fail"
      - name: Success one line angle block indicator run
        run: >
          echo "This will not fail"
```

All single line `run:` commands should error, for `|` and `>` instances, this is because we want to enforce the styling of compacting these commands into a single line if the `run:` command itself is no longer than a single line.

Reply:
I get this perspective, but what is hard to catch is both
single line
```yaml
- name: This is a test
  run: >
    echo "test"
```
and multilines

```yaml
- name: This is a test
  run: >
    echo "test"
    && echo "another test"
    && another command
```
All end up being a single line, so checking how many numbers of new lines will always return 1 (this is so because I think `yaml` does the conversion before being evaluated). I'll think about how that could be caught, let me know if you have any idea about this as well.

Reply:
I would prefer that if a single line is under the `>` or `|` it should error.  That being said, if when we parse the YAML it is giving us a single line every time after a `>`, then I wouldn't bother spending any more time on it and just let that be acceptable.

Reply:
> I would prefer that if a single line is under the `>` or `|` it should error. That being said, if when we parse the YAML it is giving us a single line every time after a `>`, then I wouldn't bother spending any more time on it and just let that be acceptable.

I agree with this. However, for consistency’s sake, should we just make that the case for `|` and `>`?

Reply:
I'm very indifferent about this @mimartin12. Should I remove the check or keep it in place, so this PR can move on

Reply:
IMO, we should get rid of both if we can't enforce both.

Reply:
Okay, will remove.